### PR TITLE
Implement V2 Client

### PR DESCRIPTION
**Pull request by boyntoni**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+1.0.0 (boyntoni)
+----------------
+
+* Add client for JW Player V2 API's
```
Comment (waxidiotic) [inline CHANGELOG.md:6]:
```suggestion
* Add client for JW Player V2 APIs
```

File: api.go [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strconv"
+)
+
+// V2Client is a light wrapper around the http.defaultClient for interacting with JW Player V2 Platform API's
```
Comment (waxidiotic) [inline api.go:13]:
```suggestion
// V2Client is a light wrapper around the http.defaultClient for interacting with JW Player V2 Platform APIs
```

**Review by waxidiotic — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -7,126 +7,44 @@ The official Go client library for accessing the [JW Platform](https://www.jwpla
...
-
-  // set URL params
-  params := url.Values{}
-  params.Set("title", "Your video title")
-  params.Set("description", "Your video description")
-
-  apiKey := os.Getenv("JWPLATFORM_API_KEY")
-  apiSecret := os.Getenv("JWPLATFORM_API_SECRET")
+jwplatform := jwplatform.New("API_SECRET")
+mySiteId := "9kzNUpe4"
```
Comment (ksindi) [inline README.md:21]: Should do `siteId` for consistency

**Review by waxidiotic — APPROVED** [review]
Hi @ksindi !

**Comment by ksindi** [comment]
Hi @waxidiotic!

File: README.md [hunk 4]
```diff
@@ -141,7 +59,6 @@ Run all tests:
 For any requests, bug or comments, please [open an issue][issues] or [submit a
 pull request][pulls].
 
-## License
+## V1 Client
 
-JW Platform API Go library is distributed under the
-[Apache v2.0 license](LICENSE).
+The V1 Client remains available for use, but is deprecated. We strongly recommend using the V2 Client. For documentation on the V1 client, please refer to the `v1` submodule.
```
Comment (ksindi) [inline README.md:64]: Minor Inconsistency "V1 Client" vs. "V1 client"

**Review by emptyflash — APPROVED** [review]

File: api.go [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+		baseURL: &url.URL{
+			Scheme: "https",
+			Host:   apiHost,
+		},
+		client: http.DefaultClient,
+	}
+}
+
+// Request performs an authenticated HTTP request to the V2 Platform API.
+func (c *V2Client) Request(method, path string, response interface{}, data interface{}, queryParams *QueryParams) error {
```
[the remark this answers is not in the document]
Comment (boyntoni) [reply api.go]: @emptyflash Just wanted to run the new solution by you, found a nice library for this

**Review by emptyflash — APPROVED** [review]
Nice, I like this query string library

**Pull request merged**